tests/acceptance/addon-smoke-test-slow.js

```diff
@@ -69,7 +72,7 @@ describe('Acceptance: addon-smoke-test', function() {
 
   it('works in most common scenarios for an example addon', function() {
     return copyFixtureFiles('addon/kitchen-sink').then(function() {
-      var packageJsonPath = addonRoot + 'package.json'
+      var packageJsonPath = path.join(addonRoot, 'package.json');
```

Comment:
I'm making this as consistent as possible. All paths are fully-qualified and normalized before being passed into anything upstream. It insulates us from future changes in a good way. For example, this was changed when it broke once the path I returned at `addonRoot` didn't have a trailing slash.